packages/teleport/src/DesktopSession/useDesktopSession.tsx

```diff
@@ -14,16 +14,42 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { useMemo } from 'react';
+import { useMemo, useEffect, useState } from 'react';
 import { getAccessToken, getHostName } from 'teleport/services/api';
 import { useParams } from 'react-router';
-import useAttempt from 'shared/hooks/useAttemptNext';
+import { Attempt } from 'shared/hooks/useAttemptNext';
 import cfg, { UrlDesktopParams } from 'teleport/config';
 import TdpClient from 'teleport/lib/tdp/client';
+import Ctx from 'teleport/teleportContext';
+import { Desktop } from 'teleport/services/desktops';
+import { stripRdpPort } from '../Desktops/DesktopList';
```

Comment:
feels a bit weird to import something from `DesktopList`, maybe have `Desktops/index.ts` re-export it?